### PR TITLE
new fuji testnet and separate mainnet version

### PR DESCRIPTION
**Pull request by ebruengwall-ant**

### CODE REVIEW

**Review by christineRPM — DISMISSED** [review]
l

**Review by gannakulikova — DISMISSED** [review]

File: validators/requirements.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ After the node registration, the node receives an assigned group of random node
 -   Ubuntu 16.04 or later LTS
 -   2TB attached storage main-net (200gb devnet)
 -   32GB RAM  
+-   8 core
```
Comment (badrogger) [inline validators/requirements.md:17]: I think we should add swap space information in this section too

**Review by badrogger — CHANGES_REQUESTED** [review]

File: validators/requirements.md (hunk 1 above)
Comment (ebruengwall-ant) [reply validators/requirements.md:17]: did you merge the pr?

File: validators/versions.md [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
+**SGX version: sgxwallet**: 1.58.2-develop.0
 
-**SKALE Manager version**: 1.5.0-develop.9
+**SKALE Manager version**: 1.5.2-develop.2
 
-**Skaled version**:  1.46.1-develop.33
+**Skaled version**:  1.46-develop.63
 
-**Skale Admin version**: 0.15.0-beta.0
+**Skale Admin version**: 0.17.0-beta.0
```
Comment (badrogger) [inline validators/versions.md]: Should be 0.17.0-beta.1

**Review by badrogger — CHANGES_REQUESTED** [review]

File: validators/requirements.md (hunk 1 above)
Comment (badrogger) [reply validators/requirements.md:17]: Oh, sorry. Need additional approve there.

File: validators/versions.md (hunk 2 above)
Comment (ebruengwall-ant) [reply validators/versions.md]: oh good find

Comment (ebruengwall-ant) [reply validators/versions.md]: Oh wait this is new ok.

**Review by badrogger — DISMISSED** [review]

**Review by christineRPM — DISMISSED** [review]

**Review by connermurphy-skale — DISMISSED** [review]

**Review by christineRPM — DISMISSED** [review]

**Review by badrogger — DISMISSED** [review]

**Review by christineRPM — APPROVED** [review]

**Review by badrogger — APPROVED** [review]

**Pull request merged**